### PR TITLE
8 split constants into secret and constants

### LINKED ISSUE #8: split constants into secret and constants
please split `constants.py` into `secrets.py` and `constants.py` for easy maintenance.

### PR DESCRIPTION
**Pull request by yhbyhb**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
-constants.py
+private.py
```
Comment (idujin) [inline .gitignore]: private.py needs to be deleted

File: csv_editor.py [hunk 2]
```diff
@@ -17,8 +17,8 @@
 logger = logging.getLogger(__name__)
 
 class CSVEditor():
-    def __init__(self):
-        self.dateman = DateManager()
+    def __init__(self, date_manager : DateManager):
```
Comment (idujin) [inline csv_editor.py:21]: Line 168 requires input arguments DateManager when creating CSVEditor

File: .gitignore (hunk 1 above)
Comment (yhbyhb) [reply .gitignore]: done

File: csv_editor.py (hunk 2 above)
Comment (yhbyhb) [reply csv_editor.py:21]: done

**Review by idujin — APPROVED** [review]
LGTM

**Pull request merged**